### ISSUE TITLE
[ADD]: Addition of various other platforms💡

### ISSUE THREAD

**Issue by AbhijithGanesh** [EddieHubCommunity/LinkFree#243]
Currently, the linkfree website only supports GitHub, twitter and YT, It would be useful if we have a broader list.

**Comment by rohan-kulkarni-25** [comment]
Yes that is true !! But I think keeping those socials which everyone will have will be better as this will maintain a consistency and let's be little professional. We should add those platfrom which will help us in reach. If we are adding some social which is not relevant and we are not active on that will be of no use.

**Comment by Vyvy-vi** [comment]
Imho, since we're aiming at an opensource free alternative to LinkTree, it would be of value if we let people add links to all of the main social websites/platforms

**Comment by voidz7** [comment]
> Imho, since we're aiming at an opensource free alternative to LinkTree, it would be of value if we let people add links to all of the main social websites/platforms

I agree with @Vyvy-vi  , I think most of the social media platforms should be added to LinkFree as we are making it an Opensource Alternative to Linktree which supports all the platforms . And also having just twitter , github and youtube would be waste of this project adding support for other  social media applications like tik-tok , Instagram , reddit would be fun and useful  .

**Comment by AbhijithGanesh** [comment]
> > Imho, since we're aiming at an opensource free alternative to LinkTree, it would be of value if we let people add links to all of the main social websites/platforms
> 
> I agree with @Vyvy-vi  , I think most of the social media platforms should be added to LinkFree as we are making it an Opensource Alternative to Linktree which supports all the platforms . And also having just twitter , github and youtube would be waste of this project adding support for other  social media applications like tik-tok , Instagram , reddit would be fun and useful  .

I totally agree with this, from the design perspective we might have same colour repeating a couple of times but I think that's okay.

**Comment by AbhijithGanesh** [comment]
I think this issue should be closed since the PR related to this was approved and a list of social media platforms have been added.

**Comment by AbhijithGanesh** [comment]
Thank you for reopening this! Are there any changes you'd like to see ? (Schmelto and other members)

**Comment by schmelto** [comment]
Reopening due the issue should closed when the PR #323 is merged

**Comment by krishguptadev** [comment]
We are supporting more of these soon after our today's poll of icon libraries.
